.github/workflows/pull_requests.yml

```diff
@@ -1,12 +1,14 @@
+name: Pull Request Target (All types)
```

Comment:
There's really no point in moving this up, but pretty much every other workflow has `name` at the top so 🤷